### DISCUSSION TITLE
how to deactivate the highlights for .text files

### DISCUSSION THREAD

**Discussion by derick-rive** [nvim-treesitter/nvim-treesitter#6121] [Q&A]
I get error when I open a query with :h , since they are .text files and the highlights tree-sitter produces errors
![Screenshot_2024-02-17_21_21_32](https://github.com/nvim-treesitter/nvim-treesitter/assets/134315002/3efd153d-5e5b-431b-818d-529bfc14054a)

When the highlight is deactivated with :TSDisable highlight
The .text file no longer produces errors, but all the color is lost in the other files
![Screenshot_2024-02-17_21_24_44](https://github.com/nvim-treesitter/nvim-treesitter/assets/134315002/94cda7b6-f43a-4fa0-a8b2-96958a4b3998)

How can I solve that????

**Comment by clason** [comment]
Fix your queries/config. Read the actual error, make sure your Neovim and nvim-treesitter (and parsers) are at the very latest version, and make sure you don't have incompatible queries installed with other plugins or colorschemes.

See #3092.
